sklearn/datasets/descr/diabetes.rst

```diff
@@ -2,7 +2,37 @@ Diabetes dataset
 
 Notes
 -----
+
 Ten baseline variables, age, sex, body mass index, average blood
 pressure, and six blood serum measurements were obtained for each of n =
 442 diabetes patients, as well as the response of interest, a
 quantitative measure of disease progression one year after baseline.
+
+Note that the 10 x variables have been standardized to have mean 0 and squared length = 1 (sum(x^2)=1).
+
+Data Set Characteristics:
+
+  :Number of Instances: 442
+
+  :Number of Attributes: First 10 columns are numeric predictive values
+
+  :Target: Column 11 is a quantitative measure of disease progression one year after baseline
+
+  :Attributes:
+    :Age:
+    :Aex:
+    :Body mass index:
+    :Average blood pressure:
+    :S1:
```

Comment:
Would it be worth using the R headers here: i.e., `tc ldl hdl tch ltg glu`, instead of `S1 S2 S3 S4 S5 S6`?  Some of those seem a bit more guessable. (I'm no medical expert, but glu, ldl and hdl look like glucose and cholesterol levels).